File: src/Application Specific/Classes/Entity Collections/ContactAttempt/ContactAttempt.cs
using System;

namespace FSM.Entity.ContactAttempts
{
    public class ContactAttempt
    {
        public ContactAttempt()
        {
        }

        public int ContactAttemptID { get; set; }
        public int ContactMethedID { get; set; }
        public int LinkID { get; set; }
        public int LinkRef { get; set; }
        public DateTime ContactMade { get; set; }
        public string Notes { get; set; }
        public int ContactMadeByUserId { get; set; }
        public string ContactMethod { get; set; }
        public string ContactMadeBy { get; set; }
    }
}

File: src/Application Specific/Classes/Entity Collections/ContactAttempt/ContactAttempt.vb
Namespace Entity.ContactAttempts

    Public Class ContactAttempt

        Public Sub New()
        End Sub

        Public Property ContactAttemptID() As Integer
        Public Property ContactMethedID() As Integer
        Public Property LinkID() As Integer
        Public Property LinkRef() As Integer
        Public Property ContactMade() As DateTime
        Public Property Notes() As String
        Public Property ContactMadeByUserId() As Integer

        Public Property ContactMethod() As String
        Public Property ContactMadeBy() As String

    End Class

End Namespace